src/Common/Common.Api/Reports/ReportsClient.cs

```diff
@@ -93,5 +103,70 @@ public IEnumerable<Import> GetImportsForWorkspace(Guid workspaceId)
         {
             return this.Client.Imports.GetImports(groupId: workspaceId.ToString()).Value?.Select(x => (Import)x);
         }
+
+        public Guid PostImport(string datasetDisplayName, string filePath)
+        {
+            using (var fileStream = new StreamReader(filePath))
+            {
+                var response = this.Client.Imports.PostImportWithFile(
+                    fileStream: fileStream.BaseStream,
+                    datasetDisplayName: datasetDisplayName,
+                    nameConflict: PowerBI.Api.V2.Models.ImportConflictHandlerMode.CreateOrOverwrite
+                );
+                return Guid.Parse(response.Id);
+            }
+        }
+
+        public Guid PostImportForWorkspace(Guid workspaceId, string datasetDisplayName, string filePath)
+        {
+            using (var fileStream = new StreamReader(filePath))
+            {
+                var response = this.Client.Imports.PostImportWithFileInGroup(
+                    groupId: workspaceId.ToString(),
+                    fileStream: fileStream.BaseStream,
+                    datasetDisplayName: datasetDisplayName,
+                    nameConflict: PowerBI.Api.V2.Models.ImportConflictHandlerMode.CreateOrOverwrite
```

Comment:
It might be better to include this as a parameter so the behavior could be configured at the cmdlet level.